resources/bin/bal.bat

```diff
@@ -91,7 +91,12 @@ if "%RUN_BALLERINA%" == "true" (
             set BALLERINA_HOME=%%a
         )
     )
-    call %CURRENT_PATH%..\distributions\!BALLERINA_HOME!\bin\ballerina.bat %*
+
+    if exist %CURRENT_PATH%..\dependencies\jdk-11.0.8+10-jre (
```

Comment:
@suganyasuven  Shouldn't we check existence of `bal`?

Reply:
Yes mistakenly checked the path.